src/pages/gateway/migration.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Migration
+description: This page describes breaking changes and how to migrate from one version of API Mesh for Adobe Developer App Builder to another.
+---
+
+# Migration
+
+This page lists breaking changes between versions of API Mesh for Adobe Developer App Builder. For a list of new features and resolved issues, refer to the [release notes](release-notes.md).
+
+To update to the latest version, run the following command:
+
+```bash
+npm install -g @adobe/aio-cli-plugin-api-mesh
+```
+
+See our [NPM page](https://www.npmjs.com/package/@adobe/aio-cli-plugin-api-mesh) or [Getting Started](getting-started.md#configure-your-environment) for more information.
+
+## Migrating to GA
+
+You will need to modify any previously created meshes to account for the following breaking changes when updating to the GA release.
+
+### Resolver Composition transform deprecated
+
+The Resolver Composition transform (`composeResolvers`) is no longer supported.
+
+Remove any instances of `composerResolvers` from your mesh before running a [create](command-reference.md#aio-api-meshcreate) or [update](command-reference.md#aio-api-meshupdate) command.
+
+### Hooks transform temporarily disabled
+
+The new [Hooks transform](hooks.md) is no longer supported.
+
+Remove any `Hook` transforms from your mesh before running a [create](command-reference.md#aio-api-meshcreate) or [update](command-reference.md#aio-api-meshupdate) command.
+
+### Mesh create and update time
+
+While most [create](command-reference.md#aio-api-meshcreate) and [update](command-reference.md#aio-api-meshupdate) commands will be instantaneous, it can take a few minutes to fully provision and propagate your mesh.
```

Comment:
create and update commands are not instantaneous. These can also take upto a few minutes

Reply:
This was changed due to Cristian's comments above:

> happen very quickly - time to execute will be instantaneous

> bare in mind that this is an async process which for sources that have big data sets can take up to minutes (5 minutes is very arbitrary - unless we want to say that''s average) since the big change is async - is important to say that this command won't error out anymore - you have to use the meshstatus for that - which you did add that info at the end of the next paragraph - but you can specify here that erroring out on this meshupdate changed